plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -32,30 +32,85 @@ namespace usbdmx {
 
 using ola::usb::LibUsbAdaptor;
 
+// "Eurolite USB-DMX512-PRO"
 const char EuroliteProFactory::EXPECTED_MANUFACTURER[] = "Eurolite";
 const char EuroliteProFactory::EXPECTED_PRODUCT[] = "Eurolite DMX512 Pro";
 const uint16_t EuroliteProFactory::PRODUCT_ID = 0xfa63;
 const uint16_t EuroliteProFactory::VENDOR_ID = 0x04d8;
 
+// "Eurolite USB-DMX512-PRO MK2" (successor device introduced in late 2016)
+const char EuroliteProFactory::EXPECTED_MANUFACTURER_MK2[] = "FTDI";
+const char EuroliteProFactory::EXPECTED_PRODUCT_MK2[] = "FT232R USB UART";
+const uint16_t EuroliteProFactory::PRODUCT_ID_MK2 = 0x6001;
+const uint16_t EuroliteProFactory::VENDOR_ID_MK2 = 0x0403;
+
+const char EuroliteProFactory::ENABLE_EUROLITE_MK2_KEY[] =
+    "enable_eurolite_mk2";
+
+EuroliteProFactory::EuroliteProFactory(ola::usb::LibUsbAdaptor *adaptor,
+                                       Preferences *preferences)
+  : BaseWidgetFactory<class EurolitePro>("EuroliteProFactory"),
+    m_adaptor(adaptor),
+    m_enable_eurolite_mk2(IsEuroliteMk2Enabled(preferences)) {
+}
+
+bool EuroliteProFactory::IsEuroliteMk2Enabled(Preferences *preferences) {
+  bool enabled;
+  if (!StringToBool(preferences->GetValue(ENABLE_EUROLITE_MK2_KEY),
+                    &enabled)) {
+    enabled = false;
+  }
+  return enabled;
+}
+
 bool EuroliteProFactory::DeviceAdded(
     WidgetObserver *observer,
     libusb_device *usb_device,
     const struct libusb_device_descriptor &descriptor) {
-  if (descriptor.idVendor != VENDOR_ID || descriptor.idProduct != PRODUCT_ID) {
-    return false;
-  }
+  bool isMK2 = false;
```

Comment:
Can we be consistent and stick with snake case here with underscores please.

Reply:
Renamed to '`is_mk2`'